client/src/store/index.js

```diff
@@ -3,13 +3,15 @@ import userReducer from './slices/userSlice/userSlice';
 import tourReducer from './slices/tourSlice/tourSlice';
 import catalogueReducer from './slices/catalogueSlice/catalogueSlice';
 import cartReducer from './slices/cartSlice/cartSlice';
+import inFavoritesSlice from './slices/inFavorites/inFavorites';
```

Comment:
please follow the naming schema
```suggestion
import inFavoritesReducer from './slices/inFavorites/inFavorites';
```

Reply:
solved